### PR TITLE
Added home and about section

### PR DESCRIPTION
**Pull request by Mithi-code**
### First Capstone project

- Used HTML

- Flex and grid for styling 

- javascript for creating feature section dynamically

### CODE REVIEW

File: About.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/about.css" />
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css" rel="stylesheet">
+    <title>Imaginary Dance</title>
+  </head>
+  <body>
+    <div class="home-section">
```
Comment (omar-labana) [inline About.html]: - [ ] The usage of `div` to contain different standalone tags like `header` and `section` has no semantic denotation (`div` tags should be used to separate/divide the content inside them), kindly remove this `div` container.

File: About.html [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+          </div>
+          <div class="menu-icons">
+            <ul>
+              <li><a href="index.html">Home</a></li>
+              <li><a href="#">Feature</a></li>
+              <li><a href="about.html">About</a></li>
+            </ul>
+          </div>
+        </nav>
+      </header>
```
Comment (omar-labana) [inline About.html]: - [ ] The `header` and `footer` tags should be identical in both pages on any screen size (currently they have different `margin`, `padding` and `font-size`).
- [ ] Please make sure that you align the `burger menu` to the `left` to match the design (currently it's to the right).
- [ ] The top dark should match the margin/padding of the given design.

Reference Design
image

Your Styling `About => 768px`
image
Your Styling `About = 320px`
image

Your Styling `Home => 768px`
image
Your Styling `Home= 320px`
image

File: About.html [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+      <footer>
+        <div class="footer-info">
+          <div>
+            <img src="./image/logo.png" alt="Logo">
+          </div>
+          <div class="footer-title"><h6>2021 Dance Meet. All Rights Reserved.</h6>
+            <p> Imaginary dance is all about bring happiness to human life. The Imaginary Dance name and logo are used under the limited license granted by world dance authorities</p>
+          </div>
+        </div>
+      </footer>
```
Comment (omar-labana) [inline About.html]: - [ ] `footer` elements should be justified between the available space, I would suggest making the `footer` a `flex` container then align the element horizontally using `justify`.
- [ ] Text inside the `footer` tag has different `font-size` on both pages, kindly make sure it's identical on the two pages.
- [ ] Please add more height to the footer in on desktop screen to match the design.

Reference Design
image

Your Styling
image

File: About.html [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+          </div>
+          <div class="partner-image">
+            <img src="./image/Zapier_logo.svg" alt="logo" />
+          </div>
+          <div class="partner-image">
+            <img src="./image/Konami_logo.svg" alt="logo" />
+          </div>
+          <div class="partner-image">
+            <img src="./image/Capcom_logo.svg" alt="logo" />
+          </div>
```
Comment (omar-labana) [inline About.html]: - [ ] According to the design guidelines, in the `desktop` screen version of the `About` page the footer has a dark background.
- [ ] According to the design guidelines, the `partners` section shouldn't be present in the `desktop` screen version of the `About` page.
Reference Design
image

File: About.html [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+            dancing, I have felt touched by something sacred. In those moments,
+            I felt my spirit soar and become one with everything that exists.”
+          </p>
+        </div>
+        <div class="home-info-2">
+          <p>
+            “You dance love, and you dance joy, and you dance dreams. And I know if I can make you smile by jumping over a couple of couches or running through a rainstorm, then I’ll be very glad to be a song and dance man.”
+          </p>
+        </div>
+      </section>
```
Comment (omar-labana) [inline About.html]: - [ ] The `header` element shouldn't be used to contain many types of text content it should be used instead as an introduction to a piece of content or section of a webpage., kindly update your `header` tag to a give more semantic meaning.
- [ ] The `hero` section (first section of the home page) should fill the whole screen view on `desktop` sized screens, the `font-size` is a bit smaller compared to the design. Kindly update the `hero` section to match the design.
- [ ] Content inside the `hero` section should be aligned a bit to the left on `desktop` screens.

Reference Design
image


Your Styling
image

File: Index.html [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+          </div>
+          <div class="home-info-2">
+            <p>
+              This is why the idea of finding love across the dance floor
+              endures – symbolizing that, when we know the true rhythm of our
+              heart, we know the other.
+            </p>
+          </div>
+        </div>
+      </section>
```
Comment (omar-labana) [inline Index.html]: - [ ] The `hero` section (first section of the home page) should fill the whole screen view on `desktop` sized screens, the `font-size` is a bit smaller compared to the design. Kindly update the `hero` section to match the design.
Reference Design
image
Your Styling
image

File: Index.html [hunk 7]
```diff
@@ -0,0 +1,200 @@
...
+      <section class="home" id="home2">
+        <div class="program-title">
+          <h2>Dance Guide</h2>
+        </div>
+        <div class="program-items">
```
Comment (omar-labana) [inline Index.html]: - [ ] Each `Card` is missing an `orange` title on both `mobile` and `desktop` views.
- [ ] Please make sure to add the missing section divider `-`, a title for each card, and make sure that individual articles have their elements organized in a vertical column, with the image at the top, title in the center, and description at the bottom on `desktop`.

Reference Design
image
Your Styling `Home => 786px`
image

File: Index.html [hunk 8]
```diff
@@ -0,0 +1,200 @@
...
+      <section class="feature-section">
+        <h2>Best Dancer's</h2>
+        <ul class="feature">
+        </ul>
+       <div class="showcontainer">
+        <div class="showMore">
+         <span> More</span>
+        </div>
+       </div>
+      </section>
```
Comment (omar-labana) [inline Index.html]: - [ ] The featured section should:
- - [ ] have the same `background-color` as the design.
- - [ ] It's currently missing the `grid` image behind each performer.
- - [ ] The page has some white space that I am not sure where it's coming from, please investigate this issue and make sure it look similar so the design.
- 
Reference Design
image

Your Styling `Home => 786px`
image

File: Index.html [hunk 9]
```diff
@@ -0,0 +1,200 @@
...
+            <img src="./image/BAFTA_logo.svg" alt="logo" />
+          </div>
+          <div class="partner-image">
+            <img src="./image/Disney_Channel_logo.svg" alt="logo" />
+          </div>
+          <div class="partner-image">
+            <img src="./image/Michael_Kors_logo.svg" alt="logo" />
+          </div>
+          <div class="partner-image">
+            <img src="./image/Zapier_logo.svg" alt="logo" />
```
Comment (omar-labana) [inline Index.html]: - [ ] Kindly, update the `background-color` of the `partners` section to match the design.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @Mithi-code, 

Excellent work so far! This is by far one of the best projects I have reviewed on this module 👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️

- [ ] The video presentation is a bit short, I would suggest recording a new one highlighting a piece of code that you find interesting or proud of making along with an overview of the project.

- [ ] The `Live Demo` link in the README is broken, please make sure to deploy your project using GitHub Pages (chane `Index.html` to `index.html` it must be small case).

- [ ] The README should contain an updated "Acknowledgements" section or similar that gives appropriate credit to Cindy Shin the author of the original design, as required by the Creative Commons License.

- [ ] The `Get Started` section in the README file is a bit unclear, I would suggest rephrasing it to a numbered list with all the instruction necessary to run the project.

- [ ] _Check the comments under the review._

- [ ] Please, consider applying and updating the requested changes on both pages so your project looks perfect. If you need any help, please reach out on slack I would be more than happy to help.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hi there 👋🏽 , 

Great work on making the changes requested by a previous reviewer 👏 
To highlight 👍🏽 
- Your cards now have the orange title.
- Your featured section has the correct background colour as the design.
- You have added the grid image behind each speaker.
- There are no white spaces on your site.
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- [ ] Like the previous reviewer mentioned, please add an acknowledgement section to your README file. In this section, give credit to the creator of the design as stated in the project requirements.
Screenshot 2021-11-18 at 10 55 14

- [ ] Your live link is still broken, please fix it as the previous reviewer stated.
- [ ] Also please update the Getting started section of your README. It should give information on how a non-technical person can clone your project and get it running.
- [ ] When I click on the about link in your desktop view, I get a 404 error as shown below, please fix it
Screenshot 2021-11-18 at 10 42 51
- [ ] In mobile view, there is a broken image when I click on the hamburger menu. I have circled it in the image below. Kindly fix it
Screenshot 2021-11-18 at 10 44 27
- [ ] When I click on any of the home or about links displayed by the hamburger menu in mobile version, I get a 404 error
Screenshot 2021-11-18 at 10 44 37
- [ ] When I click on the partners link displayed by the hamburger menu in the mobile version, I am not redirected to any page. Kindly fix all the broken links

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [ ] It would be better to give your PR a descriptive title that describes all the changes made in the project.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 10]
```diff
@@ -0,0 +1,148 @@
...
+            <a href=""><i class="fab fa-twitter"></i></a>
+          </li>
+          <li><a href="">English</a></li>
+          <li><a href="Index.html">My Page</a></li>
+          <li><a href="">Logout</a></li>
+        </ul>
+      </div>
+      <nav>
+        <div class="modal">
+          <img src="./Image/Enabled.svg" class="close" alt="closeTag" />
```
Comment (rotimiazeez) [inline about.html]: - [ ] Please note that on the about page the close button in the mobile menu is broken👀, kindly see to that👍.
screenshot (23)

File: about.html [hunk 11]
```diff
@@ -0,0 +1,148 @@
...
+              <li><a href="about.html">About</a></li>
+            </ul>
+          </div>
+        </div>
+        <div class="logo">
+          <img src="./image/logo.png" alt="logo" />
+        </div>
+        <div class="nav-hamburger">
+          <img src="./image/button.png" class="hamburger" alt="" />
+        </div>
```
Comment (rotimiazeez) [inline about.html:45]: - [OPTIONAL] Kindly fix the nav at the top of both pages😎 for a better user experience✌

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# @Mithi-code 😊, 

### Great work on making the changes requested by a previous reviewer 👏 

#### You've done well implementing some of the requested changes👍, but there are still some which aren't addressed yet🙄.
> Just a few changes and you'll be unstoppable 🚀

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — APPROVED** [review]
# Hi @Mithi-code , 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**